### ISSUE TITLE
[Bug] BadRequestHttpException for exists-rule using "dot" syntax

### ISSUE THREAD

**Issue by mroedel** [proengsoft/laravel-jsvalidation#171]
BadRequestHttpException in Validator.php line 93:
Undefined 'salutation_id' attribute

Rules:

```
return [
    'salutation.id' => 'required|exists:salutations,id'
];
```

Input:

```
<select id="salutaion_id" name="salutation[id]">...</select>
```

**Comment by web-surgeon** [comment]
I'm getting the same error trying to use distinct on an array.

BadRequestHttpException in Validator.php line 93: Undefined 'episodes_2986_original_title' attribute.

If I set 'disable_remote_validation' to true, I get a 422 response code and blank <li>'s where the error message should be.

Does distinct need to make a request if all of the data is present on the current page?

**Comment by torrentalle** [comment]
This is fixed in release `v1.4.3`
